linkchecker

```diff
@@ -669,6 +675,11 @@ if options.profile:
     do_profile = options.profile
 if options.quiet:
     config["logger"] = config.logger_new("none")
+if options.no_intro:
+    if not config["text"] or config["text"]["parts"][0] == "all":
+        config["text"]["parts"] = ["realurl", "result", "extern", "base", "name", "parenturl", "info", "warning", "dltime", "checktime", "url", "outro", "stats"]
```

Comment:
I can see why you did this but hardcoding the list of parts seems too high a price to pay.